apps/workflow-server/gcc-workflow-server/src/main/java/com/coremedia/labs/translation/gcc/workflow/GlobalLinkAction.java

```diff
@@ -128,6 +145,18 @@
    */
   private static final int FALLBACK_RETRY_COMMUNICATION_DELAY_SECS = 900;
 
+  private static final Set<String> REPOSITORY_UNAVAILABLE_ERROR_CODES = ImmutableSet.of(
```

Comment:
Ok, makes sense. 

Just maybe it would still make sense to replace it: we have replaced all usages of ImmutableList/Set/Map everywhere in CMCC code (CMS-18660) and SonarQube/SonarLint also suggests replacing it (https://rules.sonarsource.com/java/RSPEC-4738). So some future maintainer might be tempted to replace it anyway.

Reply:
Right - changed.